CODEOWNERS

```diff
@@ -13,3 +13,19 @@
 #
 
 * @fbogsany @mwear @robertlaurin @dazuma @ericmustin @arielvalentin @ahayworth @plantfansam @robbkidd
```

Comment:
Can this list be shortened to `@open-telemetry/ruby-contrib-approvers`? If so, we can probably do that on every line.

Reply:
IIRC That was by request from @fbogsany and @robertlaurin when I first set up the original codeowners file.

If there are no objections from them I will change it.

Reply:
I'm not sure if there is a reason to prefer one format over the other, but this is how collector-contrib is setup: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS. I see two benefits. When we add or remove approvers we won't have to make changes to CODEOWNERS. Also, it makes it easier to see who the supplemental code owners are.

Reply:
> I'm not sure if there is a reason to prefer one format over the other, but this is how collector-contrib is setup: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS. I see two benefits. When we add or remove approvers we won't have to make changes to CODEOWNERS. Also, it makes it easier to see who the supplemental code owners are.

I agree with @mwear on this FWIW